README.md

```diff
@@ -185,30 +189,105 @@ guarantees.
+| [Metrics SDK](./sdk/metrics)           | `opentelemetry-sdk-metrics`      | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [Trace SDK](./sdk/trace)               | `opentelemetry-sdk-trace`        | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [Log SDK](./sdk/logs)                  | `opentelemetry-sdk-logs`         | <!--VERSION_UNSTABLE-->1.14.0-alpha<!--/VERSION_UNSTABLE--> |
+| [SDK Testing](./sdk/testing)           | `opentelemetry-sdk-testing`      | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [SDK Logs Testing](./sdk/logs-testing) | `opentelemetry-sdk-logs-testing` | <!--VERSION_UNSTABLE-->1.14.0-alpha<!--/VERSION_UNSTABLE--> |
+
+### Exporter
+
+| Component                                                   | Artifact ID                                | Version                                                     |
+|-------------------------------------------------------------|--------------------------------------------|-------------------------------------------------------------|
+| [OTLP gRPC Exporter](./exporters/otlp/all)                  | `opentelemetry-exporter-otlp`              | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [OTLP gRPC Trace Exporter](./exporters/otlp/trace)          | `opentelemetry-exporter-otlp-trace`        | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [OTLP gRPC Metrics Exporter](./exporters/otlp/metrics)      | `opentelemetry-exporter-otlp-metrics`      | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [OTLP gRPC Logs Exporter](./exporters/otlp/logs)            | `opentelemetry-exporter-otlp-logs`         | <!--VERSION_UNSTABLE-->1.14.0-alpha<!--/VERSION_UNSTABLE--> |
+| [OTLP HTTP Trace Exporter](./exporters/otlp-http/trace)     | `opentelemetry-exporter-otlp-http-trace`   | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [OTLP HTTP Metrics Exporter](./exporters/otlp-http/metrics) | `opentelemetry-exporter-otlp-http-metrics` | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [OTLP HTTP Logs Exporter](./exporters/otlp-http/logs)       | `opentelemetry-exporter-otlp-http-logs`    | <!--VERSION_UNSTABLE-->1.14.0-alpha<!--/VERSION_UNSTABLE--> |
+| [OTLP Common](./exporters/otlp/common)                      | `opentelemetry-exporter-otlp-common`       | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [Jaeger gRPC Exporter](./exporters/jaeger)                  | `opentelemetry-exporter-jaeger`            | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [Jaeger Thrift Exporter](./exporters/jaeger-thrift)         | `opentelemetry-exporter-jaeger-thift`      | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [Jaeger Proto](./exporters/jaeger-proto)                    | `opentelemetry-exporter-jaeger-proto`      | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
```

Comment:
Also probably an internal component?

Reply:
Yes, and one I think can be deleted. Leaving it here for completeness until then.